addons/event/data/event_question_data.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="event_question_name" model="event.question">
```

Comment:
We can probably prevent from removing them (with an @ondelete) , so that we always have those available at hand as default values. Now that people chose them / event it is probably good to have those 3 always proposed.